### PR TITLE
feat:playlist component

### PR DESCRIPTION
**Pull request by THrapooM**
image

### CODE REVIEW

**Review by NonKhuna — CHANGES_REQUESTED** [review]
1. The color of subtitle 2 should be `text.secondary`
image

2. The bgcolor and shadow are not the same as the design. Please fix it.
image

**Comment by THrapooM** [comment]
shadow on the image?

**Comment by NonKhuna** [comment]
On The card component.

**Comment by THrapooM** [comment]
I c . I interview pap

**Comment by NonKhuna** [comment]
Good luck krub.

**Review by NonKhuna — APPROVED** [review]

File: src/components/Playlist.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import { Box, Typography, Stack } from '@mui/material'
+import Image from 'next/image'
+
+interface Props {
+  image: string
+  playlistName: string
+  creatorName: string
+}
```
Comment (karnjj) [inline src/components/Playlist.tsx:8]: add `playlistId` to props to use when redirecting the page.

**Pull request merged**